store/sqlstore/user_store.go

```diff
@@ -1474,6 +1474,14 @@ func (us SqlUserStore) AnalyticsGetInactiveUsersCount() (int64, *model.AppError)
 	return count, nil
 }
 
+func (us SqlUserStore) AnalyticsGetExternalUsers(hostDomain string) (bool, *model.AppError) {
+	count, err := us.GetReplica().SelectInt("SELECT COUNT(Id) FROM Users WHERE Email NOT LIKE :HostDomain", map[string]interface{}{"HostDomain": "%@" + hostDomain})
```

Comment:
Is it possible to use squirrel here for our SQL here (https://github.com/Masterminds/squirrel)

Reply:
Possible, but what advantages does squirrel provide in this context. In general I find it easier and more readable to write plain SQL.

Reply:
Just to keep constant and our move towards using Squirrel 
https://mattermost.atlassian.net/browse/MM-26726

Reply:
Also https://mattermost.atlassian.net/browse/MM-24946
```
Mattermost is replacing the SQL queries of the store layer to use the squirrel builder.
```